picard/util/filenaming.py

```diff
@@ -301,7 +301,12 @@ def make_short_filename(basedir, relpath, win_compat=False, relative_to=""):
     """
     # only deal with absolute paths. it saves a lot of grief,
     # and is the right thing to do, even for renames.
-    basedir = os.path.abspath(basedir)
+    try:
+        basedir = os.path.abspath(basedir)
+    except FileNotFoundError:
+        # os.path.abspath raises an exception if basedir is a relative path and
+        # cwd doesn't exist anymore
+        basedir = '/'
```

Comment:
Not sure if throwing a random `/` here is a good idea, can you explain ?

Reply:
Plus it will be invalid for Windows/

Reply:
I think we should use `Path.home()` instead, or something like that.

Reply:
IMO it should be MusicLocation from http://doc.qt.io/qt-5/qstandardpaths.html#StandardLocation-enum

Reply:
I agree, so I changed the patch to use QStandardPaths (just ignore that I wrote QStandardDirs in the commit log, I used QStandardPaths of course)